src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -574,6 +595,12 @@ function _validate_access_keys(access_key, secret_key) {
         throw_cli_error(ManageCLIError.MissingAccountAccessKeyFlag);
     }
 
+    // checking access_key and secret_key do not contain leading space
+    if (access_key !== undefined && secret_key !== undefined) {
```

Comment:
did you check in which other cases this could happen? I reported on access_key/secret_key but this could happen on other flags as well
we should add this check to all flags in my opinion

Reply:
also, how do we behave on trailing spaces?

Reply:
I think for every `string` type of flag we are facing this issue(as the value got splitted in two empty string and actual value), I have verified this for another flag `--user` getting the same issue of leading space. 
But for other types of flags if we put leading space we will get `InvalidArgumentType` error(due to empty string assignment). 
So, we should apply this check on every string type flag.

Trailing space does not create any issue here.

Reply:
We can put this check in `validate_option_type_by_value`(as before), if we want to apply this check on every string flag.

Reply:
If you move it, please remember that there are flags where the empty string is our way to unset them.

Reply:
We cannot differenciate if the empty string is provided to unset it or is it the leading space.
We need to think of another solution in that case(below 2 in my mind):
1. We are having the list of flags(that can be unset using empty string), we can remove this flags from this validation --> but after this there is no way to check leading space for this flag(that can be unset).
2. We should merge/trim the wrongly parsed flag value(because of leading space).

which approach should we follow? @romayalon @shirady @nadavMiz

Reply:
why not just add another clause to your condition checking if the whole string is not ""?

Reply:
I am not sure if I got your point but we cannot do that because when we put leading spaces then also empty string is assigned to the flag instead of whitespace. We cannot differenciate because in both case we are getting empty string here.

Reply:
Updated the code, please check. Thanks